parm/config/gfs/config.base.emc.dyn

```diff
@@ -167,7 +167,7 @@ case "${CASE}" in
         export waveGRD='glo_500'
         ;;
     "C96")
-        export OCNRES=100
+        export OCNRES=500
```

Comment:
Why not support `OCNRES=100` as well?

Reply:
It's supported, it's just not a default for any ATM resolution.

Reply:
OK, thank you for the clarification @WalterKolczynski-NOAA.

Reply:
We need to revisit this section in `config.base` as more combinations are possible, but the limitation here is the bash configuration.